backend/modules/cohorts/service.py

```
@@ -0,0 +1,577 @@
+            ))
+        
+        return simulations
+    
+    def assign_simulation_to_cohort(
+        self, cohort_id: int, simulation_data: CohortSimulationCreate,
+        user_id: int, user_role: str
+    ) -> CohortSimulationResponse:
+        """Assign a simulation to a cohort"""
+        cohort = self.repository.get_cohort_by_id(cohort_id)
+        if not cohort:
+            raise ValueError("Cohort not found")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)